### DISCUSSION TITLE
Help regarding misorientation (mis2mean) and filling data

### DISCUSSION THREAD

**Discussion by Jesusmunozmontecinos** [mtex-toolbox/mtex#656] [Ask Anything] [answered]
Dear all,

I'm Jesús, a new Mtex user and I'm facing some problems.

I want to plot an Intragranular misorientations map (attached; deired regions to fill are examples of what I want to do). There are some nonIndexed points in my map that I would like to fill using some of the smoothing functions. However, after plotting using the mis2mean function, the map remains almost the same as before applying the filters while using another function, such as .Orientations, the resulted is a nice smoothed map. How can I face this issue?. I also attached the smoothed orientations map for comparison
```matlab
[grains,ebsd('indexed').grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'));
ebsd(grains(grains.grainSize<=5)) = [];
[grains,ebsd('indexed').grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'),'angle',10*degree);

F = halfQuadraticFilter;

ebsdS = smooth(ebsd('indexed'),F,'fill',grains);

plot(ebsdS('Jadeite'),ebsdS('Jadeite').mis2mean.angle./degree)
mtexColorbar
hold on
plot(grains.boundary,'edgecolor','k','linewidth',.5)
hold off
```
![image](https://user-images.githubusercontent.com/76631957/103159953-fdfdc080-47cf-11eb-8db2-30f39082df7f.png)

![image](https://user-images.githubusercontent.com/76631957/103159980-31d8e600-47d0-11eb-86cf-91263583bf5f.png)

**Comment by kilir** [comment, marked as answer]
Hi,
After filling you need to compute mis2mean again, e.g. using calcGrains or calcGROD.
cheers Rüdiger

**Reply by Jesusmunozmontecinos** [reply]
Dear Rüdiger,
Thanks a lot for your answer, much appreciated

Cheers,
